roles/ceph-common/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: create ceph directory
+  file:
+    path: /etc/ceph
+    state: directory
```

Comment:
Alright, as long as that doesn't trip up the package install, I'm fine with this.